e2e/davinci-app/tsconfig.spec.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "moduleResolution": "NodeNext",
```

Comment:
i think this was me debugging but should be a noop since this affects test files.